test/test.js

```diff
@@ -1,3 +1,14 @@
+const dotenv = require('dotenv').config({ path: `${__dirname}/../config.env` });
 const chai = require('chai');
+const { assert } = require('chai');
+const flightController = require('../controllers/index');
+const flightsAPI = require('../controllers/getFlightsData');
 
-describe()
\ No newline at end of file
+describe('controllers/index.js', function () {
+
+    it('should remove duplicated flights', async function () {
+        const source1 = await flightsAPI.getFlights(process.env.LINK_ONE);
```

Comment:
As you are not really sure what the API is gonna return (maybe there could be repeated flights, maybe not, maybe the API could fail to response at all) its better to test with local sample data. You could create a file with some sample flights list (usually called fixtures). eg. with 2 lists that have flights (s1, s2) in common and another one that should be the expected result (r1). and them declare a test where you:
`assert.isEqual(await flightController.mergeFlightsLists(s1,s2),r1)`
We could have a call to check this together